test/test_programs/arithmetic_test/compiled_test_arithm.c

```diff
@@ -75,12 +76,36 @@ int main(int argc, char **argv) {
 
     {
         //div-zero quotient should have all bits set
-        init(++num, "DivZero");
+        init(++num, "DivZero (unsigned)");
         size_t n = 256;
```

Comment:
This is also 64-bit unsigned since size_t := uint64_t (on 64 bit architectures).
So that test is double right now. I suggest changing here to either uint32_t or unsigned (int).

Reply:
Decided to just quickly resolve this myself since I had the project loaded up and wanted to add the compiled binary anyways.